### ISSUE TITLE
Can't run on a Java project

### ISSUE THREAD

**Issue by skytreader** [geneseframework/complexity#59]
I tried to run the tool on a Java project and, unfortunately, the AST generation fails with

```
$ genese cpx -l java ./src
Welcome to @genese cli
✖ AST generation
/home/skytreader/.nvm/versions/node/v16.7.0/lib/node_modules/@genese/complexity/node_modules/@genese/core/dist/index.js:2
export * from './utils/primitives/arrays.util';
^^^^^^

SyntaxError: Unexpected token 'export'
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/skytreader/.nvm/versions/node/v16.7.0/lib/node_modules/@genese/complexity/dist/src/languages-to-json-ast/java/cstToAstCases/primary.js:5:16)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
```

I tried to add `"type": "module"` in `/home/skytreader/.nvm/versions/node/v16.7.0/lib/node_modules/@genese/complexity/package.json` [as suggested here](https://stackoverflow.com/a/40021867/777225) but to no avail. I also tried to install and run it with node v12.22.12 with pretty much the same result.

How can I run this on a Java project? Works fantastic for Typescript, btw. :) 

**EDIT (May 28 2022):** The full path of `primary.js` in the stack trace was indicating the home directory of the wrong Linux user. Edited it for clarity. (Text replacement macro to blame.)

**Comment by geneseframework** [comment]
Hello,

Thanks for your interest in @genese/complexity. Could you copy here the entire code of your file ? Or, better, could you copy a link to your project, if you have it on GitHub ? It may have some problems when you use @genese/complexity with -l java with folders containing Js or Ts files

Thanks !

**Comment by skytreader** [comment]
Sure. I was actually running it on [Enterprise Fizzbuzz](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition).

Context: I'm aware Enterprise Fizzbuzz is a joke but my use case is anything but :). I was actually in the process of writing you a longer inquiry and I was trying to use this repo as an example when I bumped into the bug described above.

**Comment by geneseframework** [comment]
There was a bug driven by an old dependency.

Your problem should now be solved with the version 1.1.29 of @genese/complexity

`npm i -g @genese/complexity`

<img width="1671" alt="image" src="https://user-images.githubusercontent.com/63037808/170860593-32a1797b-ff3c-468a-b109-016593bf26f0.png">

**Comment by skytreader** [comment]
Works perfect now, thank you. :tada:
